bridge_adaptivity/config/settings/secure_example.py

```diff
@@ -20,6 +20,11 @@
         'PASSWORD': 'bridge-adaptivity',
         'HOST': 'postgres',
         'PORT': 5432,
+        'TEST': {
+            'NAME': 'travicidb',
+            'USER': 'bridge',
+            'PASSWORD': 'bridge-adaptivity',
```

Comment:
ditto password is not changed, but HOST does please set up correct HOST variable.